tests/verifyRemoteInstances_test.js

```diff
@@ -17,24 +17,23 @@ xScenario('Open Remote Instance Page and Add mysql instances @pmm-pre-update', a
     await pmmInventoryPage.verifyAgentHasStatusRunning(mysql_service_name, version);
 });
 
-
-xScenario('Verify is the remote instances are in Running Status @pmm-post-update', async (I, adminPage, remoteInstancesPage, pmmInventoryPage) => {
-    let mysql_service_name = "mysql_remote_test";
+Scenario('Open Remote Instance Page and Add mysql instances PMM Latest @pmm-pre-update', async (I, adminPage, remoteInstancesPage, pmmInventoryPage) => {
```

Comment:
Skip this scenario, for now, we will need to add creds to Jenkins to make it work